src/Cake.DocFx.Tests/Build/DocFxBuildRunnerTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Cake.DocFx.Tests.Build
+﻿using Cake.Core.IO;
```

Comment:
@reicheltp This one is not used anyway :) Do you prefer to have usings inside or outside of namespaces? I saw that in most files they are outside,  but in some testcases which I have written they are inside. Will clean this up based on your preferences (mine is inside).